BedrockConflictManager.cpp

```diff
@@ -0,0 +1,63 @@
+#include "BedrockConflictManager.h"
+#include <libstuff/libstuff.h>
+
+BedrockConflictManager::BedrockConflictManager() {
+}
+
+void BedrockConflictManager::recordTables(const string& commandName, const set<string>& tables) {
+    {
+        lock_guard<mutex> lock(m);
+        auto commandInfo = _commandInfo.find(commandName);
```

Comment:
NAB it might be more obvious this is an interator if we called it `commandInfoIT` or something like that.